src/constants/AppConstant.ts

```diff
@@ -0,0 +1,6 @@
+export const APP_NAME = {
+    SPLASH: "Splash",
+    HOME: "Home"
+};
```

Comment:
A couple of potential improvements here
- Instead of calling it `APP_NAME`, calling it something like `SCREEN_NAMES` would make more sense. Also, keep it plural :)
- Since we are using typescript here, we can create an enum as below and use it as type as well.
```suggestion
export enum SCREEN_NAMES {
    SPLASH = "Splash",
    HOME = "Home"
};
```

Reply:
Fixed in recent commit. Thank you.